### DISCUSSION TITLE
[Accepted] CIP-1

### DISCUSSION THREAD

**Discussion by mattjoyal** [cxndao/constellation#30] [CIPs]
**Full Proposal Doc**

https://github.com/cxndao/constellation/blob/cip-1_Peer_Review/CIPs/cip-1.md
https://github.com/cxndao/constellation/discussions/30

**Goal**

Implement the use of the Constellation Improvement Proposal (CIP) as a method of DAO improvement via membership diligence and voting. 

**Description**

This CIP, first drafted by Matt Galligan, outlines & focuses on the definition, purpose, framework, and responsibilities of developing & enacting a Constellation Improvement Proposal (CIP). This will serve as the introductory 

**Benefits**

- Give Constellation DAO a framework to begin its Constellation Improvement Proposal (CIP) process.

**Point of Contact**

@mattjoyal

**Sponsors**

@galligan, @myapdx

----
**Original Post**

cip: 1,
title: CIP Purpose and Guidelines,
status: Review,
type: Governance,
author: Matt Galligan (@galligan), et al.,
created: 2022-05-25

This CIP, first drafted by Matt Galligan, outlines & focuses on the definition, purpose, framework, and responsibilities of developing & enacting a Constellation Improvement Proposal (CIP). This CIP is currently under Peer Review. Please use this Discussion as a space for review, questions, comments, changes needed. 

[CIP-1 link on GitHub](https://github.com/cxndao/constellation/blob/cip-1_Peer_Review/CIPs/cip-1.md)

**Comment by jim-stanczak** [comment]
Programs: includes improvements and changes to the DAO's software, tools, **and** documentation.

Before you begin writing a formal CIP, you _should_ vet your idea. 
=> Are these suggestions or prescriptive steps? Are there any ramifications if I skip right to proposing a CIP?

**CIP Formats and Templates** => I want to understand better why CIPs should be written in markdown format within GitHub. This is a significant point of friction that is likely to negatively impact DAO member involvement.

**Reply by mattjoyal** [reply]
Yes you are correct, the idea should be discussed, similar to this Discussion conversation, before an actual CIP is written. 

I do plan to work on a new CIP that will be focused on better explaining the _how_ and _where_ process of CIPs. Whereas, this document is focused mostly on the _who_, _what_, and _why_ of CIPs.

**Reply by mattjoyal** [reply]
As for this CIP-1, things were a bit unorganized because CIP-1 did not exist yet, thus there was no outline like this to follow. Earlier discussion had occurred in town halls and Discord before we decided that GitHub would be the place this would all occur - thus the inconsistency at the start.
